src/ApiService/ApiService/OneFuzzTypes/Model.cs

```diff
@@ -1,5 +1,7 @@
 using Microsoft.OneFuzz.Service.OneFuzzLib.Orm;
 using System;
+using System.Collections.Generic;
+using System.Net;
```

Comment:
make sure to run `dotnet format` in src\ApiService to remove those formatting issues